CMakeLists.txt

```diff
@@ -455,6 +455,11 @@ if(Legion_USE_GASNet)
   list(APPEND Legion_NETWORKS gasnet1)
 endif()
 
+option(Legion_USE_UCX "Enable the distributed UCX backend" OFF)
+if(Legion_USE_UCX)
+  list(APPEND Legion_NETWORKS ucx)
+endif()
+
```

Comment:
I thought we were moving away from this pattern. We don't even have a `gasnetex` option right now. Just configure this via `Legion_NETWORKS`.